src/uwtools/api/chgres_cube.py

```diff
@@ -25,13 +25,13 @@ def execute(
     If ``batch`` is specified, a runscript will be written and submitted to the batch system.
     Otherwise, the executable will be run directly on the current system.
 
-    :param task: The task to execute
-    :param cycle: The cycle to run
+    :param task: The task to execute.
+    :param cycle: The synoptic time.
```

Comment:
Is this an accurate/useful description of `cycle` for our use cases?

Reply:
Not particularly. Most meteorologists would take "synoptic time" to mean 00, 06, 12, and 18 UTC.

The [NOAA Glossary](https://forecast.weather.gov/glossary.php?word=synop#:~:text=Standard%20Synoptic%20Times-,The%20times%20of%200000%2C%200600%2C%201200%2C%20and%201800%20UTC,as%20the%20main%20synoptic%20times.&text=Essentially%20the%20same%20as%20mesolow.&text=Synoptic%20%2D%20relating%20to%20the%20general,features%20such%20as%20a%20thunderstorm) confirmed my generality.

Most NWP folks know what `cycle` means, but may have different synonyms when you talk to different people. Forecast ppl will call it the "start time" of the forecast. DA ppl will call it the analysis time. Verification people might call it a host of diff things based on their context.

Interestingly, NCO standards offer these descriptions:

> PDY: Date in YYYYMMDD format
> cyc: Cycle time in GMT hours, formatted HH
> cycle: Cycle time in GMT, formatted tHHz or tHHMMz

Reply:
I was trying to avoid writing `cycle: The cycle.` but will probably change to that in the absence of a clear definition for this term that isn't circular.